hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2983,5 +2983,121 @@
       stage: V2
       tests: []
     }
+    {
+      name: chip_sw_ast_concurrency
+      desc: '''Concurrency test with several AST functionalities enabled.
```

Comment:
can this particular one be merged into `max_load`?

Reply:
yeah we should be able to absorb that. let me rephrase this.

Reply:
Rephrased and combine (one testpoint less, yay!)